efficient_full_table_scan_example_code/README.md

```diff
@@ -89,3 +89,54 @@ Done!
 
 Total Execution Time: 15m32s
 ```
+
+
+
+Generic GO installation and script setup into ~/gocqlcount
+====================================================
+
+E.g. for go 1.20.4:
+
+```
+cd ~
+
+GOVERSION=go1.20.4
+curl -o $GOVERSION.linux-amd64.tar.gz https://dl.google.com/go/$GOVERSION.linux-amd64.tar.gz
+
+tar xzvf $GOVERSION.linux-amd64.tar.gz
+
+ln -s $HOME/go $HOME/$GOVERSION 
+
+cat <<EOF > go_profile.sh
+#!/bin/bash
+export GOVERSION=$GOVERSION
+export GOROOT=\$HOME/\$GOVERSION
```

Comment:
afair setting this isn't needed anymore.

Reply:
it's used for path below so you will find the binaries ...

Reply:
what I meant the variable itself isn't necessary (it used to be in older versions), you could just inline it in PATH, but both are fine for me